addons/binding/org.openhab.binding.km200/.classpath

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7"/>
+	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
+	<classpathentry kind="src" path="src/main/java"/>
+	<classpathentry kind="lib" path="lib/json-20160810.jar"/>
```

Comment:
The framework does already provide a JSON library: com.google.gson is available in version 2.7.0 and has only a slightly different API then org.json.
A change to the provided lib will reduce the size of the binding and ease maintainability since other Devs aer already familiar with the GSON lib.
wdyt?

ps: I will add an basic example in the KM200Comm.java to give you an idea of the different APIs.

Reply:
I was in holidays...
The problem is the wrong syntax on the devices api. Sometime if you ask for a float value you will get a "-NaN" as string. Almost all JSON APIs does have problems with such stuff. With this JSON lib I can handle it. It is possible in the goole json lib too?

Reply:
With the Google Gson lib this code works:

```
JsonParser parser = new JsonParser();
JsonObject root = (JsonObject) parser.parse("{ float:\"-NaN\"}");
float f = root.get("float").getAsFloat();
```

`f` will then have the valid value `NaN`.

Reply:
Hi,

ok, thanks. I will change the JSON lib.

Greetings
Markus